src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -462,5 +563,53 @@ protected void Verify_Extract(string destination, TarEntry entry, TarEntryType e
 
             AssertFileModeEquals(destination, TestPermission1);
         }
+
+        public static IEnumerable<object[]> GetNodeTarTestCaseNames()
+        {
+            foreach (string name in NodeTarTestCaseNames)
+            {
+                yield return new object[] { name };
+            }
+        }
```

Comment:
nit: we could use LINQ here

```suggestion
        => NodeTarTestCaseNames.Select(name => new object[] { name });
```